src/app/portlets/dot-templates/dot-template-list/dot-template-list.module.ts

```diff
@@ -6,10 +6,31 @@ import { DotTemplatesService } from '@services/dot-templates/dot-templates.servi
 import { DotListingDataTableModule } from '@components/dot-listing-data-table';
 import { DotPipesModule } from '@pipes/dot-pipes.module';
 import { SharedModule } from 'primeng/api';
+import { CheckboxModule } from 'primeng/checkbox';
+import { MenuModule } from 'primeng/menu';
+import { ButtonModule } from 'primeng/button';
+import { DotActionButtonModule } from '@components/_common/dot-action-button/dot-action-button.module';
+import { DotActionMenuButtonModule } from '@components/_common/dot-action-menu-button/dot-action-menu-button.module';
+import { DotAddToBundleModule } from '@components/_common/dot-add-to-bundle';
+import { DotBulkInformationModule } from '@components/_common/dot-bulk-information/dot-bulk-information.module';
+import { DialogService, DynamicDialogModule } from 'primeng/dynamicdialog';
 
 @NgModule({
     declarations: [DotTemplateListComponent],
-    imports: [CommonModule, DotListingDataTableModule, DotPipesModule, SharedModule],
-    providers: [DotTemplateListResolver, DotTemplatesService]
+    imports: [
+        CommonModule,
+        DotListingDataTableModule,
+        DotPipesModule,
```

Comment:
I'm having issues with this... if you are only using the `dm` pipe I think is better to import only that pipe module.

we might want to refactor that `DotPipesModule`